docs/learning/README.md

```diff
@@ -1,41 +1,86 @@
+   - _TIP: Once you clone the Github repo, you can `cd pocket` and run `make` from to see all the available commands._
+
+3. Get an understanding of the the V1 spec summaries by reading about the [4 modules in our docs](https://docs.pokt.network/home/learn/future). _(3-5 hours)_
+
+   - This will get you understand the spec foundations for Pocket V1
 
-    **Pro tip**: Did you know there are a bunch of `make` commands? Run `make` from the root repo to see what’s available.
+4. **Optional**: If you’re interested, you can view the [_OG v0 Pocket whitepaper_](https://pocket-network-assets.s3-us-west-2.amazonaws.com/pdfs/Pocket-Network-Whitepaper-v0.3.0.pdf). _(10-12 hours)_
 
-3.  Get an understanding of the the V1 spec summaries by reading about the 4 modules [here](https://docs.pokt.network/home/learn/future).
-4.  If you’re interested in the _OG v0 Pocket whitepaper_, check out [this link](https://pocket-network-assets.s3-us-west-2.amazonaws.com/pdfs/Pocket-Network-Whitepaper-v0.3.0.pdf).
-5.  Over time, **and not all at once**, you can start making your way through and updating the V1 specs: https://github.com/pokt-network/pocket-network-protocol.
+   - This can provide both valuable and interesting historical context into Pocket
 
-    **Tip**: We hope to publish it with V1 benchmarks on arxiv one day, so this is your change to contribute 🙂
+5. Over time, **and not all at once**, you can start making your way through and updating the [V1 specifications](https://github.com/pokt-network/pocket-network-protocol). _(30 hours)_
 
+   - Mastering these concepts won't be easy but will make you an expert on Pocket V1
+   - _TIP: We hope to publish it with V1 benchmarks on arxiv one day, so this is your change to contribute 🙂_
+   - **Optional**: If you're a core team member or heavily involved in the project, reach out to the team about getting access to the V1 specification research documents.
     <!-- For internal use only. If you're external and are reading this, reach out to the team.
        These decks from October 2021 might also help:
-       - [Utility](https://docs.google.com/presentation/d/1NU0PnegtBm5ioLu0VQMiluWT4usHnavDKrGvS3p8QdM/edit)
-       - [Persistence](https://docs.google.com/presentation/d/1qDA-pRMT1KV9byUAU49bvd_5seaILPAh6i3vA7j5l8o/edit)
-       - [P2p](https://docs.google.com/presentation/d/1CLeAcGJbM_iP76vnCoHreU1chB9vFWIYWAwQHa-MPbc/edit)
-       - [Consensus](https://docs.google.com/presentation/d/18CtSxxLLHY1N7HEJtja633mVF1_a9blaE2fe2-WgGAo/edit)
-    -->
+         - [Utility](https://docs.google.com/presentation/d/1NU0PnegtBm5ioLu0VQMiluWT4usHnavDKrGvS3p8QdM/edit)
+         - [Persistence](https://docs.google.com/presentation/d/1qDA-pRMT1KV9byUAU49bvd_5seaILPAh6i3vA7j5l8o/edit)
+         - [P2p](https://docs.google.com/presentation/d/1CLeAcGJbM_iP76vnCoHreU1chB9vFWIYWAwQHa-MPbc/edit)
+         - [Consensus](https://docs.google.com/presentation/d/18CtSxxLLHY1N7HEJtja633mVF1_a9blaE2fe2-WgGAo/edit)
+   -->
```

Comment:
Here is a just my thoughts for a more intuitive structure: (might require restructuring and analyzing which existing/listed contents mainly belong to which)

* Utility
>> Technical content 1 (related to utility)
>> Technical content 2

* Persistence
>> Technical content 1 (related to persistence)
>> Technical content 2

* P2P
>> Technical content 1 (related to P2P)
>> Technical content 2

* Consensus
>> Technical content 1 (related to Consensus)
>> Technical content 2

Reply:
Note that these are hidden (they're inside of comments) and not actually rendered on the README.

Otherwise, I'm not quite sure what the suggestion is. `Where to Start?` is meant to be a set of steps someone follows rather than a link dump of rerences.